### ISSUE TITLE
Bitte mach das fertig ich brauche sowas dringend fuer mein schul discord

### ISSUE THREAD

**Issue by Gojobriel** [Maurice037/Stundenplan-Bot#2]
wenn du gany krass bist hostest du das auch noch selber, muss aber nicht sein

**Comment by Maurice037** [comment]
Immer ruhig mit den alten Pferden :D
ich werde mal gucken ob ich die zeit finde das ganze fertig zu stellen
LG

**Comment by Gojobriel** [comment]
Natürlich, nimm dir die Zeit die du brauchst, wollte nur anmerken dass es
aktiv verfolgt wird mit großen nutzen und Interesse

**Comment by PhilDiscord** [comment]
Gibt es neues?

**Comment by Maurice037** [comment]
Ja, es wird die Tage ein update geben.  
Bin aktuell schon dran

**Comment by Gojobriel** [comment]
:POG:

**Comment by Gojobriel** [comment]
ist es inzwischen funktional?

**Comment by SDS1234** [comment]
> Ja, es wird die Tage ein update geben. Bin aktuell schon dran

Hey,
wie ist der Stand des Updates?

LG

**Comment by Maurice037** [comment]
Also, eig sollte er laufen
